rhel/SPECS/pmm-server.spec

```diff
@@ -77,6 +79,9 @@ rm -rf %{pmm_repo}-%{pmm_commit}
 
 
 %changelog
+* Fri Jun 19 2020 Andrii Skomorokhov <andrii.sokmorokhov@percona.com> - 2.7.1-1
```

Comment:
It should be 2.7.1-21 (21 is release version you changed above), and please update date

Reply:
Fixed.